extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -117,8 +117,14 @@ export class LinkerManager {
 
     if (this.allLinkerPromises_.length) {
       const navigation = Services.navigationForDoc(this.ampdoc_);
-      navigation.registerAnchorMutator(
-        this.handleAnchorMutation_.bind(this),
+      navigation.registerAnchorMutator((element, event) => {
+        if (!element.href || event.type !== 'click') {
+          return;
+        }
+        element.href = this.applyLinkers_(element.href);
+      }, Priority.ANALYTICS_LINKER);
+      navigation.registerNavigateToMutator(
+        url => this.applyLinkers_(url),
```

Comment:
Is it guaranteed to get full url here, or is relative possible?

Reply:
relative is possible. it does not affect our logic though.